components/Input.js

```diff
@@ -17,7 +17,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import styles from "styles/modules/Inputs.module.scss";
-import { useTheme } from "../hooks/useTheme";
+import { useTheme } from "providers/theme";
```

Comment:
Not a blocker on this PR, but is there a lint rule for requiring relative paths? Feels like an easy thing to miss, especially if it's added by an IDE

Reply:
True that, I have just been trying to change them as I see them come up. I can look into if there's a lint rule for it.